functions.js

```diff
@@ -0,0 +1,37 @@
+const symbols = require('./symbols');
+
+module.exports = {
+	formatOptions(options) {
+		let formatted = '';
+		let keys = Object.keys(options);
+		for (let i = 0; i < keys.length; i++) {
+			formatted += `${keys[i]} ${options[keys[i]]}\n`;
+		}
+		return formatted;
+	},
+	formatResults(results) {
+		let keys = Object.keys(results).sort(function(a,b){return results[b]-results[a]});
```

Comment:
This inline funtion should probably be an arrow function 

```js
.sort((a,b) => results[b] - results[a])
```

Reply:
Oh lol yes I'll fix that one day maybe